internal/ui/diffview.go

```
@@ -350,6 +407,269 @@ func (m *Model) reviewKey() string {
+		if !tmuxDriver.Exists(req.sess.ID) {
+			msg.err = errors.New(deadSessionHint)
+			return msg
+		}
+		if err := stor.MergeReviewState(sessID, repoRoot, req.state); err != nil {
+			msg.err = fmt.Errorf("saving review round: %w", err)
+			return msg
+		}
+		if err := tmuxDriver.SendText(req.sess.ID, req.prompt); err != nil {
+			msg.err = err
+			if rollbackErr := stor.MergeReviewState(sessID, repoRoot, req.previousState); rollbackErr != nil {
+				msg.err = fmt.Errorf("%v; restoring review drafts: %w", err, rollbackErr)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)